src/test/java/org/folio/rest/api/RequestExpirationApiTest.java

```diff
@@ -443,9 +445,7 @@ public void canExpireOpenAwaitingWithNoHoldShelfExpirationDate()
 
   private void expireRequests() throws InterruptedException, ExecutionException, TimeoutException {
     CompletableFuture<Void> expirationCompleted = new CompletableFuture<>();
-    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx(),
-      StorageTestSuite.getVertx().getOrCreateContext())
-      .setHandler(res -> expirationCompleted.complete(null));
+    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx()).setHandler(res -> expirationCompleted.complete(null));
```

Comment:
Similarly, was this automatic reformatting? I personally find the multiple line version easier to comprehend.

Reply:
i think so, reverted this